scene/animation/animation_tree.cpp

```diff
@@ -427,7 +430,7 @@ void AnimationNode::_bind_methods() {
 	}
 	BIND_VMETHOD(MethodInfo("process", PropertyInfo(Variant::REAL, "time"), PropertyInfo(Variant::BOOL, "seek")));
 	BIND_VMETHOD(MethodInfo(Variant::STRING, "get_caption"));
-	BIND_VMETHOD(MethodInfo(Variant::STRING, "has_filter"));
+	BIND_VMETHOD(MethodInfo(Variant::BOOL, "has_filter"));
```

Comment:
@godotengine/gdextension Technically, I think this is a compat breaking change. It's also an obvious fix as the signature didn't match the C++ one. So I'm not sure if we should actually merge this change for 3.5.1 (for 3.6 it's fine IMO), or keep it as broken to preserve binary compatibility?